department-of-veterans-affairs/va.gov-cms
# VAGOV-6253: Adding patch to address false 0 = empty error.

## What this does
 - When a "0" is entered in a table cell, tablefield validation function considers 0 string to be an empty value. We need to allow 0's in cells, and this patch modifies validation function to account for this.

## QA
 - Visit `node/745/edit` and open a table in the `Main Content` section. Enter a "0" in one table cell, and save, visually verifying that form errors **do not** prevent the form from submitting.
 - In same form, enter an empty value in a table cell, and save, visually verifying that form errors **do** prevent the form from submitting.
